iothub/device/src/Transport/AmqpIoT/AmqpIoTTransport.cs

```diff
@@ -200,7 +200,25 @@ private bool OnRemoteCertificateValidation(object sender, X509Certificate certif
                 return true;
             }
 
+            if (!_amqpTransportSettings.CertificateRevocationCheck && sslPolicyErrors == SslPolicyErrors.RemoteCertificateChainErrors && CausedByRevocationCheckError(chain))
```

Comment:
David, I don't test this condition "_amqpTransportSettings.CertificateRevocationCheck" as I used old sdk to test.

Reply:
I think it's OK. If `_amqpTransportSettings.CertificateRevocationCheck` is  `true` it will bypass this block and return `false` which will throw exception as expected.